### ISSUE TITLE
revised version of the Open learning Guide

### ISSUE THREAD

**Issue by johav** [OpenScienceMOOC/Module-9-Open-Educational-Resources#1]
Would it make sense to add here a revised version of the Open Learning guide by r0g_agency? 
Check https://github.com/opencultureagency/Open-Learning-Guide

We could work out a visual of the Open Science related OER available including use cases applied in various regions of the globe

**Comment by Protohedgehog** [comment]
Yes, absolutely! Thanks for this - we'll make sure to include it when production on this module is on :)

**Comment by johav** [comment]
let me make sure this get's in :P
